server/flyway/sql/V18__add_role_app_constraint_and_app_env_attr.sql

```diff
@@ -0,0 +1,49 @@
+BEGIN;
+
+-- Running upgrade V17 -> V18
+
+CREATE TABLE app_fam.fam_app_environment_type (
```

Comment:
I'm unclear why we are calling this environment_type versus just environment.

Reply:
Rename to app_fam.fam_app_environment.